src/cargo_command.rs

```diff
@@ -233,33 +233,51 @@ impl<'a> CargoBuildOutput<'a> {
         // We expect exactly one candidate; everything else is an error
         match candidates.as_slice() {
             [] => Err(err_msg("No suitable build artifacts found.")),
-            [ref the_one] => Ok(the_one),
+            [ref the_one] => Ok(vec![the_one.artifact()]),
```

Comment:
Is there a reason to still differentiate between one and several artifacts here?